src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,26 @@
+package duke;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+public class Deadline extends Task {
+
+    protected String by;
+    protected LocalDate date;
+
+    public Deadline(String description, String by) {
```

Comment:
Perhaps need a descriptive header comments for the Deadline method? 🤔
And I noticed the same issue in several other places below.